x-pack/plugins/index_management/public/application/sections/home/enrich_policies_list/policies_table/policies_table.tsx

```diff
@@ -33,43 +33,87 @@ export const PoliciesTable: FunctionComponent<Props> = ({
   onDeletePolicyClick,
   onExecutePolicyClick,
 }) => {
-  const { history } = useAppContext();
+  const {
+    history,
+    core: { capabilities },
+  } = useAppContext();
 
-  const renderToolsRight = () => {
-    return [
-      <EuiButton
-        key="reloadPolicies"
-        data-test-subj="reloadPoliciesButton"
-        iconType="refresh"
-        color="success"
-        onClick={onReloadClick}
-      >
-        <FormattedMessage
-          id="xpack.idxMgmt.enrichPolicies.table.reloadButton"
-          defaultMessage="Reload"
-        />
-      </EuiButton>,
-      <EuiButton
-        key="createPolicy"
-        fill
-        iconType="plusInCircle"
-        {...reactRouterNavigate(history, '/enrich_policies/create')}
-      >
-        <FormattedMessage
-          id="xpack.idxMgmt.enrichPolicies.table.createPolicyButton"
-          defaultMessage="Create enrich policy"
-        />
-      </EuiButton>,
-    ];
-  };
+  const createBtn = (
+    <EuiButton
+      key="createPolicy"
+      fill
+      iconType="plusInCircle"
+      data-test-subj="createPolicyButton"
+      {...reactRouterNavigate(history, '/enrich_policies/create')}
+    >
+      <FormattedMessage
+        id="xpack.idxMgmt.enrichPolicies.table.createPolicyButton"
+        defaultMessage="Create enrich policy"
+      />
+    </EuiButton>
+  );
+
+  const toolsRight = [
+    <EuiButton
+      key="reloadPolicies"
+      data-test-subj="reloadPoliciesButton"
+      iconType="refresh"
+      color="success"
+      onClick={onReloadClick}
+    >
+      <FormattedMessage
+        id="xpack.idxMgmt.enrichPolicies.table.reloadButton"
+        defaultMessage="Reload"
+      />
+    </EuiButton>,
+  ];
+
+  if (capabilities.index_management.manageEnrich) {
```

Comment:
Just a thought:

We are passing the capabilities object in quite a few places and accessing its internal keys in order to figure out if the user has the `manageEnrich` privilege. I think having a computed property in the app context (something like hasManageEnrichPrivilege) would avoid us having to specify the full path everywhere and perhaps also be a little bit more semantic than `capabilities.index_management.manageEnrich`. It might also simplify test setup as we would only need to mock a boolean instead of a full dependency.

Also, a bit unlikely I guess, but if this permission path were to change we would also have to replace it in a bunch of files rather than just once.